src/OcisResource.php

```diff
@@ -554,8 +561,35 @@ public function getContentStream()
      */
     private function getFileResponseInterface(string $fileId): ResponseInterface
     {
-        $webDavClient = new WebDavClient(['baseUri' => $this->serviceUrl . '/dav/spaces/']);
-        $webDavClient->setCustomSetting($this->connectionConfig, $this->accessToken);
+        $webDavClient = $this->createWebDavClient();
         return $webDavClient->sendRequest("GET", $fileId);
     }
+
+    /**
+     * aspect-ratio of previews to be preserved even when ask for an unusual preview size in oCIS
+     *
+     * @throws UnauthorizedException
+     * @throws ForbiddenException
+     * @throws BadRequestException
+     * @throws NotFoundException
+     * @throws InternalServerErrorException
+     * @throws InvalidResponseException
+     * @throws HttpException
+     */
+    public function getPreview(string $width, string $height): string
```

Comment:
```suggestion
    public function getPreview(int $width, int $height): string
```
we don't want the user to be able to pass `ABC` here